specification/trace/semantic_conventions/deployment.md

```diff
@@ -0,0 +1,109 @@
+
+| Value  | Description |
+|---|---|
+| `Pending` | The deployment has been accepted and is being set up |
+| `Progressing` | The deployment is currently taking place |
+| `Succeeded` | The deployment was completed correctly |
+| `Failed` | The deployment was **not** completed correctly |
+| `Unknown` | The deployment was unsuccessful and the reason is unknown |
+<!-- endsemconv -->
+
+### Workload
+
+Cloud-native applications are composed of one or several components that are usually called [workloads](https://kubernetes.io/docs/concepts/workloads/).
+Similarly to applications, each stage of the lifecycle of a workload deployment SHOULD be recorded as an Event on the span during
+which it occurred.
+
+<!-- semconv deployment.workload -->
+The event name MUST be `deployment.workload`.
+
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `deployment.workload.name` | string | The name of the single workload under deployment. | `podtato-head-hat`; `podtato-head-left-leg` | Required |
+| `deployment.workload.namespace` | string | The namespace in which the workload will be deployed. | `podtatohead` | Recommended |
+| `deployment.workload.version` | string | The version of the workload under deployment. | `v0.1.0` | Recommended |
+| `deployment.workload.status` | string | The status of the workload deployment. | `Pending` | Recommended |
+| `deployment.application.name` | string | The name of the application under deployment. | `podtato-head` | Required |
+
+`deployment.workload.status` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
+
+| Value  | Description |
+|---|---|
+| `Pending` | The deployment has been accepted and is being set up |
+| `Progressing` | The deployment is currently taking place |
+| `Succeeded` | The deployment was completed correctly |
+| `Failed` | The deployment was **not** completed correctly |
+| `Unknown` | The deployment was unsuccessful and the reason is unknown |
+<!-- endsemconv -->
+
+### Task
+
+The deployment of the new version of an application usually requires several external tasks.
+Their execution SHOULD be recorded as an Event on the span during which it occurred.
+
+<!-- semconv deployment.task -->
+The event name MUST be `deployment.task`.
+
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `deployment.task.name` | string | The name that identifies the executed task. | `database-migration`; `report-metric`; `provision-infrastructure` | Required |
+| `deployment.task.status` | string | The status of the task execution. | `Pending` | Recommended |
+| `deployment.application.name` | string | The name of the application under deployment. | `podtato-head` | Required |
```

Comment:
I don't understand the difference between workload + application here.  Can you outline that somewhere?

Reply:
Sure, maybe this should be in the "Overview" section. 
The difference is that an application is an abstraction that groups together different workloads. For example, an application could be `my-awesome-website` which comprises 3 different workloads: frontend, backend, and database. 
Tasks are actions that are executed when delivering the application and its individual workloads. For example, a task could be sending a slack message that the delivery for a specific component started/finished. 
Do you think this explanation is clear @jsuereth? If so, I would add it to the document.